### DISCUSSION TITLE
Support for import, export, backup, migrate?

### DISCUSSION THREAD

**Discussion by chrislawso** [dubinc/dub#502] [General]
Any methods that support import, export, backup, migrate?

For example if selfhost and need to move to hosted service? Or vice versa? 

How can an instance of all data be migrated to another service instance?

Thanks

**Comment by versecafe** [comment]
> Any methods that support import, export, backup, migrate?

Currently there isn't a way to migrate from self hosted to dub.co or vice versa, but the Dub API or the dub-node package would allow for a script to be written to pull and then push the data between one and the other in a sync system

@chrislawso If there is strong interest in this I can make a sync tool, or look into merging a sync feature into https://app.dub.co/settings/tokens or similar to allow full project sync using the API key from the instance you want to match.

**Comment by lukestein** [comment]
Sorry to bump two threads, but this one's related to #457.

It feels like we *really* need at least an export of some kind 🙏
